portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/SkinnableCharonPortal.java

```diff
@@ -278,6 +278,11 @@ public void doError(HttpServletRequest req, HttpServletResponse res, Session ses
 				String[] parts = getParts(req);
 				if (parts.length >= 3) {
 					String siteId = parts[2];
+					try {
+						// Lookup any alias in the URL
```

Comment:
This is general comment really, not specific to this line.  I thought we didn't want to put stuff to do with the feedback i portal ad try and keep code to do with Contact Us tool in feedback.  Have you tried doing that here?

Reply:
It's only the portal that really knows (and should know) how to un-alias a site ID. It was incorrectly putting a value into `SAKAI_PORTAL_ORIGINAL_SITEID` that wasn't a site ID.